nipoppy/workflows/pipeline.py

```diff
@@ -488,12 +488,15 @@ def submit_hpc_job(self, participants_sessions):
         except NotImplementedError as e:
             self.logger.info(
                 f"pysqa has not implemented returning the array job ID for SGE yet! "
-                f"Details: {e}"
```

Comment:
Pre-commit is complaining about `e` being defined but never used. So either we print `e` or remove `as e` on Line 488